modules/util/portal-tools-wsdd-builder/src/main/java/com/liferay/portal/tools/wsdd/builder/util/Java2WsddTask.java

```diff
@@ -175,7 +176,7 @@ private static void _addElements(
 	private static String _format(String content) throws Exception {
 		content = _stripComments(content);
 
-		SAXReader saxReader = new SAXReader();
+		SAXReader saxReader = SAXReaderFactory.getSAXReader(null, false, false);
```

Comment:
@julschong This should not be in SF. Please do the SF right after you inline the code.